README.md

```diff
@@ -6,7 +6,7 @@ This branch (master)   [![Build Status](https://dev.azure.com/cliqzci/IOS/_apis/
 
 This branch is for mainline development.
 
-This branch only works with Xcode 9.3 and supports iOS 10, and 11.
+This branch only works with Xcode 10.1 and supports iOS 10, and 11.
```

Comment:
It actually support ios 11 and above. Could you please fix this line

Reply:
@naira-cliqz 11 or 10.3? The deployment target i see is 10.3

Reply:
There is another deploment target for the target itself which is 11, and it overwrites project target. Project target comes from FF and we didn't change it to avoid one more merge conflict